Source/JavaScriptCore/wasm/WasmLLIntGenerator.cpp

```diff
@@ -1454,6 +1455,14 @@ auto LLIntGenerator::addRefFunc(uint32_t index, ExpressionType& result) -> Parti
     return { };
 }
 
+auto LLIntGenerator::addRefAsNonNull(ExpressionType reference, ExpressionType& result) -> PartialResult
+{
+    result = push();
+    WasmRefAsNonNull::emit(this, result, reference);
```

Comment:
Did we come up with a plan here yet for the bytecode overflow?

Reply:
I don't think we have a general plan yet. I've been trying to compress instructions where possible to avoid using more (e.g., `ref.test` and `ref.cast` are one LLInt opcode in an upcoming patch).